src/dlgprefautodj.cpp

```diff
@@ -38,6 +38,29 @@ DlgPrefAutoDJ::DlgPrefAutoDJ(QWidget* pParent,
     connect(autoDjIgnoreTimeEdit, SIGNAL(timeChanged(const QTime &)), this,
             SLOT(slotSetAutoDjIgnoreTime(const QTime &)));
 
+    // Auto DJ random enqueue
+    ComboBoxAutoDjRandomQueue->addItem(tr("Off"));
+    ComboBoxAutoDjRandomQueue->addItem(tr("On"));
+    ComboBoxAutoDjRandomQueue->setCurrentIndex(
+            m_pConfig->getValueString(
+                    ConfigKey("[Auto DJ]", "EnableRandomQueue"),"0").toInt());
+    // Assign a minimum value here with some logic , 5-arbitrary
```

Comment:
I'v put 5 as default here more or less randomly, 0 doesnt make sense and a relatively large intiger , I felt is better as the feature lets you enable autoDJ even when the queue is empty by adding radom tracks and adding around 5 tracks here seemed like a good start .